apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -3,72 +3,130 @@
+      <h2>{% trans 'About the course' %}</h2>
+      {% with header_level=3 %}
+        {% placeholder "course_syllabus" or %}
+        <p>{% trans 'Required' %}</p>
+        {% endplaceholder %}
+      {% endwith %}
+    </div>
+
+    <div class="course-detail__content__format">
+      <h2>{% trans 'Format' %}</h2>
+      {% placeholder "course_format" or %}
+      <p>{% trans 'Required' %}</p>
+      {% endplaceholder %}
+    </div>
+
+    <div class="course-detail__content__prerequisites">
+      <h2>{% trans 'Prerequisites' %}</h2>
+      {% placeholder "course_prerequisites" or %}
+      <p>{% trans 'Required' %}</p>
+      {% endplaceholder %}
+    </div>
+
+    <div class="course-detail__content__team">
+      <h2>{% trans 'Course team' %}</h2>
+      {% placeholder "course_team" or %}
+      <p>{% trans 'Required' %}</p>
+      {% endplaceholder %}
+    </div>
+
+    <div class="course-detail__content__plan">
+      <h2>{% trans 'Course plan' %}</h2>
+      {% placeholder "course_plan" or %}
+      <p>{% trans 'Required' %}</p>
+      {% endplaceholder %}
+    </div>
+
+      <div class="course-detail__content__subjects">
+        <h2>{% trans 'Subjects' %}</h2>
+        <ul class="course-detail__content__subjects__list">
+          {% for subject in course.subjects.all %}
+            {# If the current page is a draft, show draft subjects with a class annotation for styling #}
+            {% if current_page.publisher_is_draft %}
+              {% if subject.public_extension %}
+                <li class="course-detail__content__subjects__item">
+                  {{ subject.public_extension.extended_object.get_title }}
+                </li>
+              {% else %}
+                <li class="course-detail__content__subjects__item--draft">
+                  {{ subject.extended_object.get_title }}
+                </li>
+              {% endif %}
+            {# If the current course page is the published version, show only the subjects that are published #}
+            {% elif subject.public_extension %}
+              <li class="course-detail__content__subjects__item">
+                {{ subject.public_extension.extended_object.get_title }}
+              </li>
+            {% endif %}
+          {% empty %}
+            <li class="course-detail__content__subjects--empty">
+              {% trans "No associated subjects" %}
             </li>
-          {% endif %}
-        {# If the current page is the published version, show only the organizations that are published #}
-        {% elif organization.public_extension %}
-          <li class="course-detail__organizations__item">
-            {{ organization.public_extension.extended_object.get_title }}
-          </li>
-        {% endif %}
-      {% endfor %}
-    </ul>
-
-    <div class="course-detail__active-session">
+          {% endfor %}
+        </ul>
+      </div>
+
+      <div class="course-detail__content__organizations">
```

Comment:
I think you need to display first the main organization (see course.organization_main)